src/PKSim.CLI.Core/Services/ExportSimulationRunner.cs

```diff
@@ -102,14 +107,15 @@ public async Task<SimulationExport> ExportSimulation(Simulation simulation, Expo
       {
          var projectName = project.Name;
          var simulationName = simulation.Name;
-         var simulationFolder = Path.Combine(exportRunOptions.OutputFolder, FileHelper.RemoveIllegalCharactersFrom(simulationName));
+         var escapedSimulationName = FileHelper.RemoveIllegalCharactersFrom(simulationName);
+         var simulationFolder = Path.Combine(exportRunOptions.OutputFolder, escapedSimulationName);
          DirectoryHelper.CreateDirectory(simulationFolder);
 
          var simulationExport = new SimulationExport
          {
             Project = projectName,
             SimulationFolder = simulationFolder,
-            Simulation = simulationName
+            Simulation = escapedSimulationName
```

Comment:
RE was doing Folder\Simulation-Result.csv to get the results. IT did nto work with non escaped stuff obviously